content/contribute/_index.md

```diff
@@ -50,35 +48,35 @@ Now you can build and run BotKube by one of the following ways
    a. Using helm (v3)
 
    ```sh
-   $ helm repo add infracloudio https://infracloudio.github.io/charts
-   $ helm repo update
-   $ kubectl create namespace botkube
-   $ helm install --version v9.99.9-dev botkube --namespace botkube \
-     --set communications.slack.enabled=true \
-     --set communications.slack.channel=<SLACK_CHANNEL_NAME> \
-     --set communications.slack.token=<SLACK_API_TOKEN_FOR_THE_BOT> \
-     --set settings.clustername=<CLUSTER_NAME> \
-     --set settings.kubectl.enabled=<ALLOW_KUBECTL> \
-     --set image.repository=<your_account>/botkube \
-     --set image.tag=latest \
-     infracloudio/botkube
+   helm repo add botkube https://infracloudio.github.io/charts

```

Comment:
So we don't change Helm chart repo for now? 🤔

Reply:
nope, we don't have anything in place, will need to think about this replacement and run a POC with OCI